ee/api/test/__snapshots__/test_instance_settings.ambr

```diff
@@ -0,0 +1,8 @@
+# name: TestInstanceSettings.test_update_recordings_ttl_setting
```

Comment:
I can't figure out why the snapshot doesn't add the `ALTER TABLE` query that actually updates the TTL, any ideas?

Reply:
that's because snapshot only captures select queries! https://github.com/PostHog/posthog/blob/master/ee/clickhouse/util.py#L37

Reply:
(which can be overridden)

Reply:
One decent way around this: Create a new mixin with the `capture_select_queries` function that is inherited first so the method resolution order gives it precedence over the ClickhouseTestMixin.

Reply:
Your way is much nicer 🙌